examples/voip/Server.cpp

```diff
@@ -133,11 +133,9 @@ class NetworkAudioStream : public sf::SoundStream
                 // (so we protect any operation on it with the mutex)
                 {
                     const std::lock_guard lock(m_mutex);
-                    const std::size_t     oldSize = m_samples.size();
-                    m_samples.resize(oldSize + sampleCount);
```

Comment:
Might want to replace that with a `reserve` call

Reply:
`std::vector::insert` will allocate more space if needed.

Reply:
I guess it's fine when we specify a begin/end range. For an InputIterator there doesn't seem to be a guarantee